app/views/shared/_statements_table.html.haml

```diff
@@ -9,7 +9,7 @@
         %th= Statement.human_attribute_name(:sent_to)
         - if local_assigns[:admin_view]
           %th= Account.model_name.human
-        - else
+        - if cross_facility?
```

Comment:
This is not directly related to this ticket, and probably should have been caught in the my statements stories. Can we add the account number to the My Accounts tab to it looks like the transactions/members tabs?

![screen shot 2016-10-27 at 3 53 27 pm](https://cloud.githubusercontent.com/assets/1099111/19785042/960f8476-9c5d-11e6-97df-0fce46eeb071.png)
![screen shot 2016-10-27 at 3 53 19 pm](https://cloud.githubusercontent.com/assets/1099111/19785043/9612d392-9c5d-11e6-9fb9-dc38c44aa04c.png)